src/structure/3dmol/index.d.ts

```diff
@@ -178,12 +178,12 @@ export interface AtomStyleSpec {
     cross: Partial<CrossStyleSpec>;
     /** draw bonds as capped cylinders */
     stick: Partial<StickStyleSpec>;
-    /** draw atoms as spheres */
-    sphere: Partial<SphereStyleSpec>;
+    /** draw atom bodies */
+    body: Partial<BodyStyleSpec>;
```

Comment:
This file should only contains types exported from 3Dmol, as defined here: https://3dmol.csb.pitt.edu/doc/types.html#toc14, nothing else.

We don't get to choose how 3dmol interprets this data =)